lib/tasks/create_data_types.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
Migth be worth adding an arbitrary test for the tasks just to ensure that we catch any code drift that might break the task. I have added them in my branch just a test to ensure that stuff is created. Also worth adding instructions in the readme.

Reply:
Also worth adding some feedback once the task has completed.